src/Microsoft.Health.Dicom.Core/Features/Validation/LongStringValidation.cs

```diff
@@ -15,12 +15,12 @@ namespace Microsoft.Health.Dicom.Core.Features.Validation;
 /// </summary>
 internal class LongStringValidation : IElementValidation
 {
-    public void Validate(DicomElement dicomElement, bool withLeniency = false)
+    public void Validate(DicomElement dicomElement, ValidationStyle validationStyle = ValidationStyle.Strict)
     {
         string value = dicomElement.GetFirstValueOrDefault<string>();
-        if (withLeniency)
+        if (validationStyle == ValidationStyle.Default)
```

Comment:
Do you think it will be more cleaner to write an extension method for DicomElement like `GetValue(ValidationStyle validationStyle). So in all the places we just call `dicomElement.GetValue(validationStyle)`